### PR TITLE
chore: update owlbot java image

### PR DESCRIPTION
**Pull request by Neenu1995**

### CODE REVIEW

**Review by eaball35 — APPROVED** [review]

**Review by lesv — APPROVED** [review]

**Comment by Neenu1995** [comment]
Waiting until tomorrow to merge this. Trying out a different renovate configuration on this repo. This PR puts it back.

**Pull request closed without merging**
